test/e2e/build_test.go

```diff
@@ -308,6 +308,30 @@ RUN exit 5`, ALPINE)
 		Expect(data).To(ContainSubstring(buildah.Version))
 	})
 
+	It("podman build and check identity with always", func() {
+		// with --pull=always
+		session := podmanTest.Podman([]string{"build", "--pull=always", "-f", "build/basicalpine/Containerfile.path", "--no-cache", "-t", "test1", "build/basicalpine"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		// Verify that OS and Arch are being set
+		inspect := podmanTest.Podman([]string{"image", "inspect", "--format", "{{ index .Config.Labels }}", "test1"})
+		inspect.WaitWithDefaultTimeout()
+		data := inspect.OutputToString()
+		Expect(data).To(ContainSubstring(buildah.Version))
+
+		// with --pull-always
+		session := podmanTest.Podman([]string{"build", "--pull-always", "-f", "build/basicalpine/Containerfile.path", "--no-cache", "-t", "test2", "build/basicalpine"})
```

Comment:
`inspect` and `session` are already declared so other two lines just need `=` instead of `:=`. I guess that's why CI is failing